recipes/openturns/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "1.7" %}
+{% set variant = "openblas" %}
+
+package:
+  name: openturns
+  version: {{ version }}
+
+source:
+  fn: openturns-{{ version }}.tar.gz
+  url: https://github.com/openturns/openturns/archive/v{{ version }}.tar.gz
+  sha1: 9535120721b20a45460538f3ed8fa0f0fec1d797
+# see https://github.com/openturns/openturns/pull/206:
+  patches:
+    - openturns-config-path.patch
+
+build:
+  number: 200
+# build timeouts when jobs are not split:
+  skip: true  # [py3k]
+  skip: true  # [win]
```

Comment:
@jakirkham the py3k line will be deleted in production as build jobs are split when using conda-smithy instead of staged-recipes and we have time to build all flavors